explainer.md

```diff
@@ -690,13 +706,16 @@ enum VRFrameOfReferenceType {
 
 interface VRFrameOfReference : VRCoordinateSystem {
   readonly attribute VRStageBounds? bounds;
+  attribute EventHandler onboundschange;
```

Comment:
I wondered about the same thing myself, but I figured that a `VRCoordinateSystemEvent` would be a bit more futureproof. Especially given that Javascript will duck-type the hell out of these things for us. ;)

I don't see the need to provide the bounds in the event, since it's just as easy to access them off the frame of reference. There's not a lot of difference between `event.coordinateSystem.bounds` and `event.bounds`.